validation/validator.go

```diff
@@ -200,6 +202,20 @@ func (v *successValidator) ValidateParameterLargeDuration(prmName string, value
 	return v
 }
 
+func (v *successValidator) ValidateParameterBase64(prmName string, value *string, mandatory bool) Validator {
+	if value == nil {
+		if mandatory {
+			return &failedValidator{err: cerrors.CreateMissingParameterError(prmName)}
+		}
+	} else {
+		if _, err := base64.StdEncoding.DecodeString(*value); err != nil {
+			return &failedValidator{err: cerrors.CreateBadRequestError(cerrors.MsgErrInvalidParam + "." + prmName)}
```

Comment:
I guess that a regexp would be more performant... or only use len(*value)%4 last characters

Reply:
Found these regex:
Allow empty strings: ^(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?$
Disallow empty strings: ^(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=|[A-Za-z0-9+/]{4})$

Reply:
It seems those regexp are not fully valid according some stackoverflow comments. As performance is not critical for this, at least for now, let's go for the safest path :)